index.bs

```diff
@@ -2015,14 +1988,107 @@ Issue: This needs to be generalised to work with realms too.
 <div algorithm="remote end steps for session.unsubscribe">
 The [=remote end steps=] with |session| and |command parameters| are:
 
-1. Let the |list of event names| be the value of the <code>events</code> field of
-   |command parameters|.
+1. If |command parameters| matches the <code>session.UnsubscribeByAttributesRequest</code> production:
+
+   1. Let |event names| be a [=/set=] of |command parameters|["<code>events</code>"].
+
+   1. Let |top-level traversable context ids| be a [=/set=].
```

Comment:
The same issue with wrong navigable IDs to be interpreted as a global subscription.